cmd/commands/common.go

```diff
@@ -360,44 +356,21 @@ func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *strin
 	return nil
 }
 
-func getIngressHostFromUserInput(cmd *cobra.Command, ingressHost *string) error {
-	if ingressHost != nil && *ingressHost != "" {
-		*ingressHost = strings.TrimSpace(*ingressHost)
-		return nil
-	}
-
-	if store.Get().Silent {
-		return fmt.Errorf("missing ingress host")
-	}
-
-	ingressHostPrompt := promptui.Prompt{
-		Label: "Ingress host (required)",
-	}
-
-	ingressHostInput, err := ingressHostPrompt.Run()
+func getIngressHostFromCluster(ctx context.Context, opts *RuntimeInstallOptions) error {
+	fmt.Print("Retrieving ingress controller info from your cluster...\n")
```

Comment:
use logger if you want this message to print to the user (in debug or info). no need for fmt.Print